pkg/driver/efs_watch_dog.go

```diff
@@ -39,22 +95,43 @@ type execWatchdog struct {
 	execArg []string
 	// the cmd that is running
 	cmd *exec.Cmd
+	// efs-utils config file path
+	efsUtilsCfgPath string
 	// stopCh indicates if it should be stopped
 	stopCh chan struct{}
 
 	mu sync.Mutex
 }
 
-func newExecWatchdog(cmd string, arg ...string) Watchdog {
+func newExecWatchdog(efsUtilsCfgPath, cmd string, arg ...string) Watchdog {
 	return &execWatchdog{
-		execCmd: cmd,
-		execArg: arg,
-		stopCh:  make(chan struct{}),
+		efsUtilsCfgPath: efsUtilsCfgPath,
+		execCmd:         cmd,
+		execArg:         arg,
+		stopCh:          make(chan struct{}),
 	}
 }
 
-func (w *execWatchdog) start() {
+func (w *execWatchdog) start() error {
+	if err := w.updateConfig(); err != nil {
+		return err
+	}
+
 	go w.runLoop(w.stopCh)
+
+	return nil
+}
+
+func (w *execWatchdog) updateConfig() error {
+	efsUtilsConfig := template.Must(template.New("efs-utils-config").Parse(efsUtilsConfigTemplate))
+	f, err := os.Create(w.efsUtilsCfgPath)
+	if err != nil {
+		return fmt.Errorf("cannot create config file %s for efs-utils. Error: %v", w.efsUtilsCfgPath, err)
+	}
+	if err = efsUtilsConfig.Execute(f, efsUtilsConfig); err != nil {
```

Comment:
Why pass `efsUtilsConfig` in as the template object ? Does the template have any substitution variable?

Reply:
Not in this PR. In a upcoming PR, I'll customize the client info section. Also in the future we may set the region in the config file as well.